src/azure-cli/azure/cli/command_modules/backup/custom_wl.py

```diff
@@ -49,7 +49,8 @@
                      'SAPHANA': 'SAPHanaDatabase',
                      'SQLDataBase': 'SQLDataBase',
                      'SAPHanaDatabase': 'SAPHanaDatabase',
-                     'SAPASE': 'SAPAseDatabase'}
+                     'SAPASE': 'SAPAseDatabase',
+                     'SAPAseDatabase': 'SAPAseDatabase'}
```

Comment:
Do we need to add this to `protectable_item_type_map` as well? AFAIK SAPASE comes under the Azure Workload backup management type, so it would be required so that some operations (list protectable items, show protectable instance) don't fail.